docs/data/pages.ts

```diff
@@ -148,62 +148,98 @@ const pages: MuiPage[] = [
+            title: 'Date Time Field',
+            newFeature: true,
+          },
+        ],
       },
       {
-        pathname: '/x/react-date-pickers/fields',
-        title: 'Field components',
+        pathname: '/x/react-date-pickers/date-range-picker',
+        title: 'Date Range components',
+        plan: 'pro',
         scopePathnames: [
-          '/x/react-date-pickers/fields',
-          '/x/react-date-pickers/date-field',
-          '/x/react-date-pickers/time-field',
-          '/x/react-date-pickers/date-time-field',
+          '/x/react-date-pickers/date-range-picker',
           '/x/react-date-pickers/date-range-field',
-          '/x/react-date-pickers/time-range-field',
-          '/x/react-date-pickers/date-time-range-field',
         ],
-        newFeature: true,
         children: [
-          { pathname: '/x/react-date-pickers/fields', title: 'Getting started' },
-          { pathname: '/x/react-date-pickers/date-field', title: 'Date Field' },
-          { pathname: '/x/react-date-pickers/time-field', title: 'Time Field' },
-          { pathname: '/x/react-date-pickers/date-time-field', title: 'Date Time Field' },
+          { pathname: '/x/react-date-pickers/date-range-picker', title: 'Date Range Picker' },
           {
             pathname: '/x/react-date-pickers/date-range-field',
             title: 'Date Range Field',
-            plan: 'pro',
+            newFeature: true,
           },
+        ],
+      },
+      {
+        pathname: '/x/react-date-pickers/time-range-picker',
+        title: 'Time Range components',
+        plan: 'pro',
+        scopePathnames: [
+          '/x/react-date-pickers/time-range-picker',
+          '/x/react-date-pickers/time-range-field',
+        ],
+        children: [
+          { pathname: '/x/react-date-pickers/time-range-picker', title: 'Time Range Picker 🚧' },
           {
             pathname: '/x/react-date-pickers/time-range-field',
             title: 'Time Range Field',
-            plan: 'pro',
+            newFeature: true,
+          },
+        ],
+      },
+      {
+        pathname: '/x/react-date-pickers/date-time-range-picker',
+        title: 'Date Time Range components',
+        plan: 'pro',
+        scopePathnames: [
+          '/x/react-date-pickers/date-time-range-picker',
+          '/x/react-date-pickers/date-time-range-field',
+        ],
+        children: [
+          {
+            pathname: '/x/react-date-pickers/date-time-range-picker',
+            title: 'Date Time Range Picker 🚧',
           },
           {
             pathname: '/x/react-date-pickers/date-time-range-field',
             title: 'Date Time Range Field',
-            plan: 'pro',
+            newFeature: true,
           },
         ],
       },
+      { pathname: '/x/react-date-pickers/validation' },
+      { pathname: '/x/react-date-pickers/localization' },
+      { pathname: '/x/react-date-pickers/fields', title: 'Keyboard editing' },
```

Comment:
I'm wondering if those should not be on top of the components, because by priority of reading, I think it's:

1. Common usage (validation, localization)
2. Specific usage (Date picker special props)
3. Customization (slots)
4. components API

Reply:
IMHO—the current order makes a tiny bit more sense.
From DX PoV I'd firstly be looking at which component to (can I) use. And only after that consider localisation and validation. 🤔 

The only section to me that sounds and looks off is the `Keyboard editing`... 🙈 

I think that we should either update the page title/contents a bit or use a different name.

Maybe going with `Field components` would make more sense?
And, IMO, this section could use a `newFeature: true` prop 🤔

Reply:
With the new grouping by type, I feel like we are pushing the user to first find its component page and use it as the entry-point for the rest.

I don't think you come to the doc looking for information about the Localization / Validation before first checking how to use the component you need.

On nuance, the Localization pages will contain very basic topic on how to install and use the adapters after #6625 (which is probably even more early-stage in the user needs), but the needed sections will be linked from the getting started.

Reply:
The only section to me that sounds and looks off is the Keyboard editing... see_no_evil

I think that we should either update the page title/contents a bit or use a different name.

Maybe going with Field components would make more sense?
And, IMO, this section could use a newFeature: true prop thinking

____

@LukasTy 

This page contains two quite different kind of content

1. Basic topics about what fields are. This could probably be moved to the *Getting Started* page on a new sections explaining that we have 3 type of components: the *Pickers*, the *Fields* and the *Calendar* / *Clock*. It could also be in a *Components* page that would be a summary of the available components if we feel that it's to much for the *Getting Started*/

2. Advanced topics (the current *Advanced* section)? This could be moved in an *Advanced* folder like on the grid with a *Field behaviors* page. Or if we don't want to create a folder, directly at the root.

I had to put the current page somewhere but I do agree that it is not great.
I can name it *Field components* (as the alpha only have power users who can take the time to look what "Field" mean) and work on a better split on a follow up PR.

Reply:
Yes, I agree that we can simply name it `Field components` for now and re-work the whole structure and content in a follow-up PR leaving this one with single responsibility. 😉